### ISSUE TITLE
Allow range block in aci_vlan_pool

### ISSUE THREAD

**Issue by vfiftyfive** [CiscoDevNet/terraform-provider-aci#173]
Add support for block definition of ranges in vlan pool and remove requirement for "vlan-" prefix before vlan number:

```
resource "aci_vlan_pool" "my_pool" {
  name = "my_vlan_pool"
  alloc_mode = "dynamic"
  range {
    from  = "100"
    to = "200"
}
  range {
    from  = "1500"
    to = "2000"
}
```

**Comment by lhercot** [comment]
I think you can use aci_ranges to achieve the same. Do we still need to add this capability?

**Comment by trenzy** [comment]
You can use aci_ranges for this, See below. It still requires "vlan-xyz" for the the definition of the start and ending ranges.

# Add VLAN Ranges to VLAN pool with static allocation
resource "aci_ranges" "vlan_pool" {
  description = "Created with Terraform v0.13.4"
  vlan_pool_dn  = aci_vlan_pool.tf-vlan-pool.id
  from  = "vlan-650"
  to  = "vlan-659"
  alloc_mode = "static"
  role = "external"
}

**Comment by vfiftyfive** [comment]
I don't think that it achieves the same thing. A block allows you multiple ranges. For aci_ranges, you'd have to create multiple resources, which is what the block usage is trying to avoid.

Nic

**Comment by trenzy** [comment]
Would this capability be better in aci_ranges instead of aci_vlan_pool, since this is where we define the vlan ranges we want?

Then again, I'm not sure why we have separate resources for this.
